### ISSUE TITLE
Could not find a valid gem libxml-ruby-x64-mingw32 for Ruby 3.0 on Windows

### ISSUE THREAD

**Issue by soundasleep** [xml4r/libxml-ruby#176]
The README says:

> If you are running Windows, then install the libxml-ruby-x64-mingw32 gem. 

However under Ruby 3.0 I can't seem to find this gem:

```
C:\path>gem install libxml-ruby-x64-mingw32
ERROR:  Could not find a valid gem 'libxml-ruby-x64-mingw32' (>= 0) in any repository
```

I assume this is because that gem is built with Ruby 2.7. 

Can the README be edited to say "Ruby 2.7 only" or (preferably) a Ruby 3.0 gem is released of `libxml-ruby-x64-mingw32`?

**Comment by MSP-Greg** [comment]
If you have MSYS2 tools installed at C:\msys64, you should be able to install it with:
```
ridk enable
pacman -S mingw-w64-x86_64-libxml2
gem install libxml-ruby --platform=ruby -- --with-opt-include=C:/msys64/mingw64/include/libxml2
```

For the soon to be released Ruby 3.1
```
ridk enable
pacman -S mingw-w64-ucrt-x86_64-libxml2
gem install libxml-ruby --platform=ruby -- --with-opt-include=C:/msys64/ucrt64/include/libxml2
```

**Comment by cfis** [comment]
I just released version 3.2.2 which packages Ruby 3.0 and Ruby 3.1 versions.
